libs/platform/src/lib/components/form/datetime-picker/datetime-picker.component.spec.ts

```diff
@@ -173,4 +173,9 @@ describe('PlatformDatetimePickerComponent', () => {
 
         expect(inputGroupEl.nativeElement.classList.contains('is-error')).toBeTrue();
     });
+
+    it('should take 100% of container width', async () => {
```

Comment:
Is this part safe without passing and calling `done` method?

Reply:
We didn't actually need the `async` function for this test, so have removed it. I was not aware of the `done` method, but from what I read here, `async` and `done` are just different ways of handling asynchronous code, so both should be valid.